### DISCUSSION TITLE
Bounding Box Discussion

### DISCUSSION THREAD

**Discussion by cooldelay** [openTCS/opentcs#167] [Q&A] [answered]
We want to verify issues related to the bounding box and its associated rights of way. However, we encountered some difficulties. As shown in Figure
![影像](https://github.com/user-attachments/assets/842b627b-a953-44b2-8a30-fab81912adbd)
, when setting defaultrouter.shortestpath.edgeEvaluators to BOUNDING_BOX, DISTANCE and configuring the vehicle's bounding box to be 1000mm in length and width (note that each line on the map is 1000mm), the vehicle operates normally. 
![影像 (3)](https://github.com/user-attachments/assets/c75225e0-3dff-496e-8376-d05f3cf7a0e6)
![影像 (4)](https://github.com/user-attachments/assets/cbba1928-fd17-4e18-81dc-5b62752dc84c)
However, when the bounding box is set to 1000mm in length and 1100mm in width, the vehicle fails to move.
![影像 (1)](https://github.com/user-attachments/assets/c2a6b7cc-b8d2-4ab4-94ac-b9ed19058288)
![影像 (2)](https://github.com/user-attachments/assets/6b2fbac5-631f-49ba-9192-9e2ec4d21a87)
Could anyone provide guidance on how to properly configure and utilize the bounding box functionality? What could be the reasons for the vehicle's inability to move when the bounding box dimensions are changed?

**Comment by swltr** [comment, marked as answer]
@cooldelay What are the allowed bounding boxes on the points set to? The default is (1000, 1000, 1000); a vehicle with a larger bounding box would not fit in there and would thus not be routed to/across such a point.

**Reply by cooldelay** [reply]
Thanks. Sorry, I didn't configure the bounding box size for the points.
